### PR TITLE
add liquibase icon, update gradle

### PR DESCRIPTION
**Pull request by mateusz-bajorek**

### CODE REVIEW

File: gradlew.bat [hunk 1]
```diff
@@ -1,100 +1,103 @@
-@rem
```
Comment (mateusz-bajorek) [inline gradlew.bat]: Looks like new gradle wrapper version has different line ending characters for bat file.

Comment (jonathanlermitage) [reply gradlew.bat]: my bad, we should create a `.gitattributes` file with something like  
```
gradlew text eol=lf
gradlew.bat text eol=crlf
```  
this way, line endings will be OK on git pull.

Comment (mateusz-bajorek) [reply gradlew.bat]: Should I amend this in that case?

Comment (jonathanlermitage) [reply gradlew.bat]: yes, thx.

Comment (mateusz-bajorek) [reply gradlew.bat]: Done 👍

**Comment by jonathanlermitage** [comment]
Thx!

**Comment by jonathanlermitage** [comment]
PR is merged, but GitHub did not close it.  
Closing manually.

**Pull request closed without merging**